src/test/scala/com/softwaremill/realworld/users/UserEndpointTestSupport.scala

```diff
@@ -147,7 +168,143 @@ object UserEndpointTestSupport:
       callRegisterUser(authorizationHeader, uri, userRegisterData)
     )(isLeft(equalTo(HttpError("{\"error\":\"User with username jake already in use\"}", sttp.model.StatusCode(409)))))
 
-  def checkIfInvalidPasswordErrorOccur(
+  def checkIfUserEmailAlreadyExistsErrorOccurInUpdate(
+      authorizationHeader: Map[String, String],
+      uri: Uri,
+      userUpdateData: UserUpdateData
+  ): ZIO[UsersServerEndpoints, Throwable, TestResult] =
+    assertZIO(
+      callUpdateUser(authorizationHeader, uri, userUpdateData)
+    )(isLeft(equalTo(HttpError("{\"error\":\"User with email john@example.com already in use\"}", sttp.model.StatusCode(409)))))
+
+  def checkIfUserUsernameAlreadyExistsErrorOccurInUpdate(
+      authorizationHeader: Map[String, String],
+      uri: Uri,
+      userUpdateData: UserUpdateData
+  ): ZIO[UsersServerEndpoints, Throwable, TestResult] =
+    assertZIO(
+      callUpdateUser(authorizationHeader, uri, userUpdateData)
+    )(isLeft(equalTo(HttpError("{\"error\":\"User with username john already in use\"}", sttp.model.StatusCode(409)))))
+
+  def checkIfEmptyFieldsErrorOccurInRegister(
+      authorizationHeader: Map[String, String],
+      uri: Uri,
+      userRegisterData: UserRegisterData
+  ): ZIO[UsersServerEndpoints, Throwable, TestResult] =
+    assertZIO(
+      callRegisterUser(authorizationHeader, uri, userRegisterData)
+    )(
+      isLeft(
+        equalTo(
+          HttpError(
+            "{\"errors\":{\"body\":[\"Invalid value for: body (" +
```

Comment:
**suggestion:** Use triple quotes to avoid escaping quotes in JSON

```scala
""" {"errors": "body": ["Invalid value for: body (
//...
"""

Reply:
With triple quotes im not able to show this string in the readable form, because i have to use "enter" inside this string and this cause that my test fail.

Reply:
I used triple quotes but i had to use additional: 
```scala 
"...".stripMargin.replaceAll("\\n", "") 
```

Reply:
Nice, thanks!